spacewalk/config/etc/httpd/conf.d/public.conf

```
@@ -0,0 +1,10 @@
+Alias /public /srv/www/htdocs/pub
+<Directory "/srv/www/htdocs/pub">
+    SetEnv VIRTUALENV
+    Options Indexes
+    Require all granted
+    AllowOverride All
+</Directory>
+
+RewriteEngine On
+RewriteRule ^/pub/(.+)$ /public/$1  [L,PT]
```

I think we can simplify this file since we are redirecting all to this folder and only have 
```suggestion
Alias /pub /srv/www/htdocs/pub
<Directory "/srv/www/htdocs/pub">
    SetEnv VIRTUALENV
    Options Indexes
    Require all granted
    AllowOverride All
</Directory>
```